docker/release.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
If we commit this with the executable bit, you won't need `bash` — just `./release.sh`...